disnake/enums.py

```diff
@@ -299,1205 +277,175 @@ def __setattr__(self, name: str, value: Any) -> None:
-    """An auto moderation rule was updated.
-
-    When this is the action, the type of :attr:`~AuditLogEntry.target` is
-    the :class:`AutoModRule` or :class:`Object` with the ID of the auto moderation rule which
-    was updated.
-
-    Possible attributes for :class:`AuditLogDiff`:
-
-    - :attr:`~AuditLogDiff.name`
-    - :attr:`~AuditLogDiff.enabled`
-    - :attr:`~AuditLogDiff.trigger_type`
-    - :attr:`~AuditLogDiff.event_type`
-    - :attr:`~AuditLogDiff.actions`
-    - :attr:`~AuditLogDiff.trigger_metadata`
-    - :attr:`~AuditLogDiff.exempt_roles`
-    - :attr:`~AuditLogDiff.exempt_channels`
-
-    .. versionadded:: 2.6
-    """
     automod_rule_delete                   = 142
-    """An auto moderation rule was deleted.
-
-    When this is the action, the type of :attr:`~AuditLogEntry.target` is
-    the :class:`Object` with the ID of the auto moderation rule which
-    was deleted.
-
-    Possible attributes for :class:`AuditLogDiff`:
-
-    - :attr:`~AuditLogDiff.name`
-    - :attr:`~AuditLogDiff.enabled`
-    - :attr:`~AuditLogDiff.trigger_type`
-    - :attr:`~AuditLogDiff.event_type`
-    - :attr:`~AuditLogDiff.actions`
-    - :attr:`~AuditLogDiff.trigger_metadata`
-    - :attr:`~AuditLogDiff.exempt_roles`
-    - :attr:`~AuditLogDiff.exempt_channels`
-
-    .. versionadded:: 2.6
-    """
     automod_block_message                 = 143
-    """A message was blocked by an auto moderation rule.
-
-    When this is the action, the type of :attr:`~AuditLogEntry.target` is
-    the :class:`Member` or :class:`User` who had their message blocked.
-
-    When this is the action, the type of :attr:`~AuditLogEntry.extra` is
-    set to an unspecified proxy object with these attributes:
-
-    - ``channel``: A :class:`~abc.GuildChannel`, :class:`Thread` or :class:`Object` with the channel ID where the message got blocked.
-    - ``rule_name``: A :class:`str` with the name of the rule that matched.
-    - ``rule_trigger_type``: A :class:`AutoModTriggerType` value with the trigger type of the rule.
-    """
     automod_send_alert_message            = 144
-    """An alert message was sent by an auto moderation rule.
-
-    When this is the action, the type of :attr:`~AuditLogEntry.target` is
-    the :class:`Member` or :class:`User` who had their message flagged.
-
-    See :attr:`automod_block_message` for more information on how the
-    :attr:`~AuditLogEntry.extra` field is set.
-    """
     automod_timeout                       = 145
-    """A user was timed out by an auto moderation rule.
-
-    When this is the action, the type of :attr:`~AuditLogEntry.target` is
-    the :class:`Member` or :class:`User` who was timed out.
-
-    See :attr:`automod_block_message` for more information on how the
-    :attr:`~AuditLogEntry.extra` field is set.
-    """
     # fmt: on
 
     @property
     def category(self) -> Optional[AuditLogActionCategory]:
-        """The category of this :class:`AuditLogAction`."""
```

Comment:
.

Reply:
Resolved in 606392a3f3350c8f1e22fa43c4b268855841d5cf.